### PR TITLE
Bookstore css

### PR DESCRIPTION
**Pull request by tingamapuro04**
I have worked on the bookstore-css branch where I have followed the zeplin style guidelines to style my component. I have only altered the form section since the zeplin project, we only have two input fields one for the book title and the other for the category but I don't want to hard code the author values, I chose to include one more input field for the book author.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -13,19 +15,37 @@ function Book({ itemId, title, author }) {
...
+            </div>
+          </div>
+          <div className="vertical-separator">
+            <p className="try">|</p>
+          </div>
+          <div className="Book-last">
+            <p className="current">CURRENT CHAPTER</p>
+            <p className="Chapter">Chapter 10</p>
+            <button className="Update" type="button">UPDATE PROGRESS</button>
           </div>
```
Comment (roshan-bajgain) [inline src/components/Book.js:47]: - [ ] If you see the design then the navigation panel is at the top page covering the whole width. See the image below for reference.
### your design
hhhhhhhhhh

### Design provided by Microverse
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @tingamapuro04, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️


## Required Changes ♻️

- [ ] Beside this everything is fine

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
I have worked on the changes requested by reducing the margin at the top, left, and right so that my design can resemble the one on zeplin

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @tingamapuro04 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**